src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.Core.cs

```diff
@@ -73,6 +80,10 @@ private object DeserializeObject(int typeIndex)
         }
 
         // Issue https://github.com/dotnet/runtime/issues/39290 tracks finding an alternative to BinaryFormatter
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2060:MakeGenericMethod",
```

Comment:
This suppression can be removed now, since this method is marked with `RequiresUnreferencedCode`.